src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Sdk.Razor.CurrentVersion.targets

```diff
@@ -10,6 +10,14 @@ Copyright (c) .NET Foundation. All rights reserved.
 ***********************************************************************************************
 -->
 <Project ToolsVersion="14.0" TreatAsLocalProperty="_RazorSdkTasksTFM;_Targeting30OrNewerRazorLangVersion">
+
+  <Import Project="$(MSBuildThisFileDirectory)Sdk.Razor.StaticAssets.ProjectSystem.targets"
```

Comment:
Project="Sdk.Razor.StaticAssets.ProjectSystem.props"

Reply:
This file is gone